typings/tiny-warning.d.ts

```diff
@@ -0,0 +1,5 @@
+declare module 'tiny-warning' {
```

Comment:
https://github.com/alexreardon/tiny-warning/blob/master/package.json#L11

Reply:
Nice, I guess I have to update the packages. I only looked in the DefinetlyTyped repo for typings, and they didn’t exist there 🙈